prometheus-to-sd/main.go

```diff
@@ -64,9 +65,10 @@ var (
 		"Cluster location to use. If not set, defaults to value from GCE Metadata Server.")
 	projectOverride = flag.String("project-id", "",
 		"GCP project to send metrics to. If not set, defaults to value from GCE Metadata Server.")
-	monitoredResourceTypes = flag.String("monitored-resource-types", "gke_container",
-		"The monitored resource types to use, either the legacy 'gke_container', or the new 'k8s'")
-	omitComponentName = flag.Bool("omit-component-name", true,
+	// monitoredResourceTypes = flag.String("monitored-resource-types", "gke_container", "The monitored resource types to use, either the legacy 'gke_container', or the new 'k8s'")
+	monitoredResourceTypePrefix = flag.String("monitored-resource-type-prefix", "", "MonitoredResource type prefix, to be appended by 'container', 'pod', and 'node'.")
+	nodeLabelsOverride          = flag.String("node-labels-override", "", "Manually specified MonitoredResource labels for nodes. It is in URL parameter format, like 'A=B&C=D&E=F'. When this field is specified, 'monitored-resource-type-prefix' is also required.")
```

Comment:
Why are you calling this override? As far as I can tell, this is not overriding, it's completely replacing them. How about static/custom/extra labels? "Node" isn't accurate either - this can be attached to pod or container metrics as well.

Reply:
Agree, that it's very confusing. Maybe monitoredResourceLabels?

Reply:
PTAL.